pca_cca.go

```diff
@@ -136,7 +119,7 @@ func (c CC) Right(spheredSpace bool) (vecs *mat64.Dense) {
 //
 // Canonical correlation analysis finds associations between two sets of
 // variables on the same observations by finding linear combinations of the two
-// sphered datasets that maximise the correlation between them.
+// sphered datasets that maximize the correlation between them.
```

Comment:
I'm completely consistent here - internal comments are written in English my be, user-facing documentation is written in American.

The price of the benefits of cultural emperialism are what they are :)